easybuild/easyconfigs/s/SolexaQA++/SolexaQA++-3.1.5-foss-2016b.eb

```diff
@@ -0,0 +1,35 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Adam Huffman
+# The Francis Crick Institute
+easyblock = 'MakeCp'
+
+name = 'SolexaQA++'
+version = '3.1.5'
+
+homepage = 'http://solexaqa.sourceforge.net'
+description = """SolexaQA calculates sequence quality statistics and creates visual representations
+ of data quality for second-generation sequencing data. Originally developed for the Illumina system
+ (historically known as “Solexa”), SolexaQA now also supports Ion Torrent and 454 data."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+sources = ['%(name)s_v%(version)s.zip']
+source_urls = ['https://sourceforge.net/projects/solexaqa/files/src/']
+
+dependencies = [
+    ('Boost', '1.61.0'),
+    ('zlib', '1.2.8'),
+]
+
+start_dir = 'source'
+
+files_to_copy = [ '%(name)s' ]
+
+sanity_check_paths = {
+    'files': ['%(name)s'],
+    'dirs': ["."]
```

Comment:
@verdurin please don't use `.` here, that's cheating ;)

fixed in https://github.com/verdurin/easybuild-easyconfigs/pull/4